src/platform/posix/ipc.c

```diff
@@ -30,7 +30,7 @@ extern uint8_t *posix_fuzz_buf, posix_fuzz_sz;
 // thinking/experimentation about how much fuzzing we want to do at a
 // time...
 static uint8_t fuzz_in[65536];
-static uint8_t fuzz_in_sz;
+static size_t fuzz_in_sz;
```

Comment:
Nitpick: this bug is actually pretty subtle and maybe deserves more explanation in the commit message, if for no better reason than to sell how clever a catch it was.

Reply:
lol fair